packages/theme/package.json

```diff
@@ -21,6 +21,7 @@
     "start": "dev -- start-storybook -p 6006"
   },
   "dependencies": {
+    "@masknet/icons": "workspace:*",
```

Comment:
Please also add this package as a project reference in tsconfig.

Reply:
Fixed. Thanks.